src/file/stat.cr

```diff
@@ -93,8 +112,10 @@ class File
       io << ", rdev=0x"
       rdev.to_s(16, io)
       io << ", size=" << size
-      io << ", blksize=" << blksize
-      io << ", blocks=" << blocks
+      {% unless flag?(:win32) %}
```

Comment:
Maybe add a comment saying that it's ignored on windows because those getter are currently not implemented and would raise?